content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -86,34 +86,35 @@ For a client application, your game must implement the following use cases to ma
 - `JoinSession`
 - `LeaveSession`
 
-The AWS GameLift Gem provides both [C++ APIs](usage/cpp-api/) and [Scripting](usage/scripting/) to support developers. You can implement these use cases using either method. 
+The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [scripting](scripting/). You can implement these use cases using either method.
+
 
 ## 4. Integrate server
 
-To establish communication between your server and Amazon GameLift service, you must notify Amazon GameLift that your server is ready and then respond to GameLift notifications.
+To establish communication between your server and GameLift, you must notify GameLift that your server is ready, and then have your server respond to GameLift notifications.
 
-For more details on Server Notifications, read [C++ API for AWS GameLift Gem](usage/cpp-api/)
+For more details on GameLift server notifications, refer to the [Server notifications](cpp-api/#server-notifications) section of the AWS GameLift Gem C++ API page.
 
 ## 5. Set up AWS credentials
 
 To work with AWS resources in O3DE you must set up AWS credentials for your users. 
 
-For more details, read [Configuring AWS Credentials for O3DE](/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials/) in the AWS Core Gem section.
+For more details, refer to [Configuring AWS Credentials](/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials/).
 
 {{< note >}} 
 This step is only required for developers who perform remote tests and infrastructure builds against the Amazon GameLift service.
 
-Alternatively, you can test on your local machine using GameLift Local, which does not require AWS credentials. To test against GameLift Local, there is support for the client override. For more information, read [AWS GameLift Gem Local Testing](/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing/). 
+Alternatively, you can test on your local machine using GameLift Local, which does not require AWS credentials. To test against GameLift Local, there is support for the client override. For more information, refer to [AWS GameLift Gem Local Testing](/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing/). 
 {{< /note >}} 
 
 
-## 6. Set up AWS CLI and CDK
---------------------------
+## 6. Set up the AWS CLI and AWS CDK
 
-The AWS GameLift provides a sample CDK application that you can use to model and deploy your server to the Amazon GameLift service. To do this, you must have [AWS CLI](https://aws.amazon.com/cli/) and [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/) installed on your local machine.
+The AWS GameLift Gem provides a sample AWS Cloud Development Kit (AWS CDK) application that you can use to model and deploy your server to GameLift. To do this, you must have the [AWS Command Line Interface (AWS CLI)](https://aws.amazon.com/cli/) and [AWS CDK](https://aws.amazon.com/cdk/) installed on your local machine.
```

Comment:
minor: "....that you can use to model Amazon GameLift resources and deploy your server"